win32/game_of_life/game_of_life.odin

```diff
@@ -0,0 +1,436 @@
+package game_of_life
+
+/*********************************************************************
+                            GAME  OF  LIFE
+                            (using win32)
+
+This example shows a simple setup for a game with Input processing,
+updating game state and drawing game state to the screen.
+
+You can
+* Left-Click to bring a cell alive
+* Right-Click to kill a cell
+* Press <Space> to (un)pause the game
+* Press <Esc> to close the game
+
+The game starts paused.
+
+Build with:
+
+odin build . -resource:game_of_life.rc
+
+**********************************************************************/
+
+import "base:intrinsics"
+import "base:runtime"
+import "core:fmt"
+import "core:math/rand"
+import "core:os"
+import win32 "core:sys/windows"
+import "core:time"
+
+// aliases
+L :: intrinsics.constant_utf16_cstring
+color :: [4]u8
+int2 :: [2]i32
+
+// constants
+TITLE :: "Game Of Life"
+WORLD_SIZE: int2 : {128, 64}
+ZOOM :: 12 // pixel size
+FPS :: 20
+
+BLACK :: color{0, 0, 0, 255}
+WHITE :: color{255, 255, 255, 255}
+
+HELP_TEXT :: `ESC - Quit
+P - Toggle Pause
+H - Toggle Help`
+help_rect: win32.RECT = {20, 20, 160, 100}
+show_help := true
+
+IDT_TIMER1: win32.UINT_PTR : 10001
+
+color_bits :: 1
+palette_count :: 1 << color_bits
+color_palette :: [palette_count]color
+
+bitmap_info :: struct {
+	bmiHeader: win32.BITMAPINFOHEADER,
+	bmiColors: color_palette,
+}
+
+screen_buffer :: [^]u8
+
+ConfigFlag :: enum u32 {
```

Comment:
Related to my previous comment about naming. This is also a type definition but uses another naming style.

It should probably be consistent and I think it should be `Config_Flag` if you want to be like the other core libs.

Reply:
I tired to change the types to Pascal_Snake_Case, helped make them standout so good suggestion :)